app/Admin/src/Http/Resources/Auth/AdminResource.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Admin\Http\Resources\Auth;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class AdminResource extends JsonResource
```

Comment:
It may make sense to rename this to `AuthenticatedAdminResource` to differentiate it and use it only during the login / sign in process.

Also, don't forget to return the token.